doc/users/prev_whats_new/whats_new_3.3.0.rst

```diff
@@ -304,43 +415,156 @@ clabels has been changed to (``2 + zorder`` passed to `~.axes.Axes.contour` /
 `~.axes.Axes.contourf`).
 
 
+Better control of ``Axes.pie`` normalization
+--------------------------------------------
+
+Previously, `.Axes.pie` would normalize its input *x* if ``sum(x) > 1``, but
+would do nothing if the sum were less than 1. This can be confusing, so an
+explicit keyword argument *normalize* has been added. By default, the old
+behavior is preserved.
+
+By passing *normalize*, one can explicitly control whether any rescaling takes
+place and whether partial pies should be created.
+
+
 Simple syntax to select fonts by absolute path
 ----------------------------------------------
 
 Fonts can now be selected by passing an absolute `pathlib.Path` to the *font*
 keyword argument of `.Text`.
 
 
-Add generalized "mathtext.fallback" rcParam
--------------------------------------------
+Improved font weight detection
+------------------------------
+
+Matplotlib is now better able to determine the weight of fonts from their
+metadata, allowing to differentiate between fonts within the same family more
+accurately.
+
+
+rcParams improvements
+---------------------
+
+``matplotlib.rc_context`` is now a ``contextlib.contextmanager``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`matplotlib.rc_context` can now be used as a decorator (technically, it is now
+implemented as a `contextlib.contextmanager`), e.g., ::
```

Comment:
so can it also be used as a context manager? do you want to mention both examples than?

Reply:
It has always been usable as a context manager. The `contextlib.contextmanager` in the title confused me too, so maybe it's so important to mention it explicitly.